interface/code_systems/dataloads_ajax.php

```diff
@@ -177,7 +177,7 @@
                         <div >
                         </div>
                     </div>
-                    <div class="wrpr col-md-4 col-sm-7 offset-sm-1">
+                    <div class="wrpr col-md-7 col-sm-7 offset-sm-1">
                         <div class="stg_dets">
```

Comment:
Hi @yashrajbothra 

Nice work looks good, just only one place, I guess @tywrenn didn't like the ` offset-sm-1"` maybe will request to remove it.

Reply:
@yashrajbothra Yep gotta remove the offsets

Reply:
How do we give space without offset :(

**Without Offset**
![image](https://user-images.githubusercontent.com/47683556/87680831-885ee480-c79b-11ea-9751-64a065c3346b.png)

**With Offset**
![image](https://user-images.githubusercontent.com/47683556/87680968-b17f7500-c79b-11ea-915b-a2892e9dcf3f.png)

Reply:
I guess you can add some padding or margin?

Reply:
If you're talking about the gray background, you can just add it to the row class and it will work fine.

Reply:
added margin b/w both cols. Looks okay?